app/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
+import './index.css';
```

Comment:
How come this was added in?

Reply:
The `index.css` file is in the master branch and reordered in this PR when I made corrections to the dependency ordering. If css files should be higher up than the App component, let me know and I'll make the change. Thanks @adrianmcli !